libs/novui/src/global.styles.ts

```diff
@@ -14,6 +15,11 @@ export const GLOBAL_CSS = defineGlobalStyles({
 
     backgroundColor: 'surface.page',
     overflow: 'hidden',
+    scrollbarWidth: 'thin',
+    scrollbarColor: {
+      base: `${LEGACY_COLOR_TOKENS.legacy.B80.value} transparent`,
+      _dark: `${LEGACY_COLOR_TOKENS.legacy.B30.value} transparent`,
```

Comment:
Panda doesn't support token evaluation for CSS values requiring 2 or more properties to be specified inside a single string and doesn't have native support for these properties ([ref](https://panda-css.com/docs/utilities/interactivity)), so we need to use template literals instead.